Logless Middleware & Types from @types/express

When installing the express type definitions using:

```
$npm install @types/express
```

I get a compile error in logless.d.ts when I attempt to convert from TS to JS:

```
ERROR in /Users/michaelmyers/Desktop/mortgage/node_modules/bespoken-tools/lib/logless/logless.d.ts
(1,32): error TS2307: Cannot find module '~express/lib/router/index'.

ERROR in /Users/michaelmyers/Desktop/mortgage/node_modules/bespoken-tools/lib/logless/logless.d.ts
(2,30): error TS2307: Cannot find module '~express/lib/express'.
```

If I use the same type definitions in the BST project, it works:

```
  "dependencies": {
    "express": "registry:npm/express#4.14.0+20160925001530"
  }
```
This is most likely a discrepancy between the definition files.

--- a/docs/api/classes/alexacontext.html
+++ b/docs/api/classes/alexacontext.html
@@ -122,7 +122,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L16">alexa/alexa-context.ts:16</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L16">alexa/alexa-context.ts:16</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -157,7 +157,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L50">alexa/alexa-context.ts:50</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L50">alexa/alexa-context.ts:50</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">string</span></h4>
@@ -174,7 +174,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L78">alexa/alexa-context.ts:78</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L78">alexa/alexa-context.ts:78</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">boolean</span></h4>
@@ -191,7 +191,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L23">alexa/alexa-context.ts:23</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L23">alexa/alexa-context.ts:23</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">string</span></h4>
@@ -208,7 +208,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L58">alexa/alexa-context.ts:58</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L58">alexa/alexa-context.ts:58</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">AudioPlayer</span></h4>
@@ -225,7 +225,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L62">alexa/alexa-context.ts:62</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L62">alexa/alexa-context.ts:62</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">boolean</span></h4>
@@ -242,7 +242,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L74">alexa/alexa-context.ts:74</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L74">alexa/alexa-context.ts:74</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">void</span></h4>
@@ -259,7 +259,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L35">alexa/alexa-context.ts:35</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L35">alexa/alexa-context.ts:35</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">InteractionModel</span></h4>
@@ -276,7 +276,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L66">alexa/alexa-context.ts:66</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L66">alexa/alexa-context.ts:66</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">void</span></h4>
@@ -293,7 +293,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L70">alexa/alexa-context.ts:70</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L70">alexa/alexa-context.ts:70</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <a href="alexasession.html" class="tsd-signature-type">AlexaSession</a></h4>
@@ -310,7 +310,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L54">alexa/alexa-context.ts:54</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L54">alexa/alexa-context.ts:54</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -333,7 +333,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L46">alexa/alexa-context.ts:46</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L46">alexa/alexa-context.ts:46</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -356,7 +356,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L31">alexa/alexa-context.ts:31</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L31">alexa/alexa-context.ts:31</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">string</span></h4>
@@ -373,7 +373,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-context.ts#L39">alexa/alexa-context.ts:39</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-context.ts#L39">alexa/alexa-context.ts:39</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">string</span></h4>

--- a/docs/api/classes/alexasession.html
+++ b/docs/api/classes/alexasession.html
@@ -113,7 +113,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-session.ts#L9">alexa/alexa-session.ts:9</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-session.ts#L9">alexa/alexa-session.ts:9</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <a href="alexasession.html" class="tsd-signature-type">AlexaSession</a></h4>
@@ -133,7 +133,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-session.ts#L17">alexa/alexa-session.ts:17</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-session.ts#L17">alexa/alexa-session.ts:17</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">object</span></h4>
@@ -155,7 +155,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-session.ts#L28">alexa/alexa-session.ts:28</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-session.ts#L28">alexa/alexa-session.ts:28</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">string</span></h4>
@@ -172,7 +172,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-session.ts#L36">alexa/alexa-session.ts:36</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-session.ts#L36">alexa/alexa-session.ts:36</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">boolean</span></h4>
@@ -189,7 +189,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-session.ts#L32">alexa/alexa-session.ts:32</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-session.ts#L32">alexa/alexa-session.ts:32</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -212,7 +212,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-session.ts#L21">alexa/alexa-session.ts:21</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-session.ts#L21">alexa/alexa-session.ts:21</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -235,7 +235,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/alexa-session.ts#L40">alexa/alexa-session.ts:40</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/alexa-session.ts#L40">alexa/alexa-session.ts:40</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">void</span></h4>

--- a/docs/api/classes/audioitem.html
+++ b/docs/api/classes/audioitem.html
@@ -115,7 +115,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/audio-item.ts#L7">alexa/audio-item.ts:7</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/audio-item.ts#L7">alexa/audio-item.ts:7</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -137,7 +137,7 @@
 					<div class="tsd-signature tsd-kind-icon">stream<span class="tsd-signature-symbol">:</span> <a href="audioitemstream.html" class="tsd-signature-type">AudioItemStream</a></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/audio-item.ts#L7">alexa/audio-item.ts:7</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/audio-item.ts#L7">alexa/audio-item.ts:7</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -154,7 +154,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/audio-item.ts#L20">alexa/audio-item.ts:20</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/audio-item.ts#L20">alexa/audio-item.ts:20</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">

--- a/docs/api/classes/audioitemstream.html
+++ b/docs/api/classes/audioitemstream.html
@@ -94,7 +94,7 @@
 					<div class="tsd-signature tsd-kind-icon">expected<wbr>Previous<wbr>Token<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span><span class="tsd-signature-symbol"> =&nbsp;null</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/audio-item.ts#L28">alexa/audio-item.ts:28</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/audio-item.ts#L28">alexa/audio-item.ts:28</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -104,7 +104,7 @@
 					<div class="tsd-signature tsd-kind-icon">offset<wbr>InMilliseconds<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">number</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/audio-item.ts#L29">alexa/audio-item.ts:29</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/audio-item.ts#L29">alexa/audio-item.ts:29</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -114,7 +114,7 @@
 					<div class="tsd-signature tsd-kind-icon">token<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span><span class="tsd-signature-symbol"> =&nbsp;null</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/audio-item.ts#L27">alexa/audio-item.ts:27</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/audio-item.ts#L27">alexa/audio-item.ts:27</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -124,7 +124,7 @@
 					<div class="tsd-signature tsd-kind-icon">url<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span><span class="tsd-signature-symbol"> =&nbsp;null</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/alexa/audio-item.ts#L26">alexa/audio-item.ts:26</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/alexa/audio-item.ts#L26">alexa/audio-item.ts:26</a></li>
 						</ul>
 					</aside>
 				</section>

--- a/docs/api/classes/bstalexa.html
+++ b/docs/api/classes/bstalexa.html
@@ -128,7 +128,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L58">client/bst-alexa.ts:58</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L58">client/bst-alexa.ts:58</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -184,7 +184,7 @@
 					<div class="tsd-signature tsd-kind-icon">Default<wbr>Intent<wbr>Schema<wbr>Location<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span><span class="tsd-signature-symbol"> =&nbsp;&quot;speechAssets/IntentSchema.json&quot;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L56">client/bst-alexa.ts:56</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L56">client/bst-alexa.ts:56</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -194,7 +194,7 @@
 					<div class="tsd-signature tsd-kind-icon">Default<wbr>Sample<wbr>Utterances<wbr>Location<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span><span class="tsd-signature-symbol"> =&nbsp;&quot;speechAssets/SampleUtterances.txt&quot;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L57">client/bst-alexa.ts:57</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L57">client/bst-alexa.ts:57</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -211,7 +211,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L101">client/bst-alexa.ts:101</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L101">client/bst-alexa.ts:101</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <a href="alexacontext.html" class="tsd-signature-type">AlexaContext</a></h4>
@@ -228,7 +228,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L200">client/bst-alexa.ts:200</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L200">client/bst-alexa.ts:200</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -301,7 +301,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L214">client/bst-alexa.ts:214</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L214">client/bst-alexa.ts:214</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -357,7 +357,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L117">client/bst-alexa.ts:117</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L117">client/bst-alexa.ts:117</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -392,7 +392,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L147">client/bst-alexa.ts:147</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L147">client/bst-alexa.ts:147</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -427,7 +427,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L241">client/bst-alexa.ts:241</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L241">client/bst-alexa.ts:241</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -483,7 +483,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L256">client/bst-alexa.ts:256</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L256">client/bst-alexa.ts:256</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -539,7 +539,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L287">client/bst-alexa.ts:287</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L287">client/bst-alexa.ts:287</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -569,7 +569,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L271">client/bst-alexa.ts:271</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L271">client/bst-alexa.ts:271</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -625,7 +625,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L225">client/bst-alexa.ts:225</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L225">client/bst-alexa.ts:225</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -687,7 +687,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L184">client/bst-alexa.ts:184</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L184">client/bst-alexa.ts:184</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -753,7 +753,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L89">client/bst-alexa.ts:89</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L89">client/bst-alexa.ts:89</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -802,7 +802,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L297">client/bst-alexa.ts:297</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L297">client/bst-alexa.ts:297</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">

--- a/docs/api/classes/bstalexaevents.html
+++ b/docs/api/classes/bstalexaevents.html
@@ -95,7 +95,7 @@
 					<div class="tsd-signature tsd-kind-icon">Audio<wbr>Player<wbr>Playback<wbr>Finished<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span><span class="tsd-signature-symbol"> =&nbsp;&quot;AudioPlayer.PlaybackFinished&quot;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L13">client/bst-alexa.ts:13</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L13">client/bst-alexa.ts:13</a></li>
 						</ul>
 					</aside>
 					<div class="tsd-comment tsd-typography">
@@ -111,7 +111,7 @@
 					<div class="tsd-signature tsd-kind-icon">Audio<wbr>Player<wbr>Playback<wbr>Nearly<wbr>Finished<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span><span class="tsd-signature-symbol"> =&nbsp;&quot;AudioPlayer.PlaybackNearlyFinished&quot;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L20">client/bst-alexa.ts:20</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L20">client/bst-alexa.ts:20</a></li>
 						</ul>
 					</aside>
 					<div class="tsd-comment tsd-typography">
@@ -127,7 +127,7 @@
 					<div class="tsd-signature tsd-kind-icon">Audio<wbr>Player<wbr>Playback<wbr>Started<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span><span class="tsd-signature-symbol"> =&nbsp;&quot;AudioPlayer.PlaybackStarted&quot;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L27">client/bst-alexa.ts:27</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L27">client/bst-alexa.ts:27</a></li>
 						</ul>
 					</aside>
 					<div class="tsd-comment tsd-typography">
@@ -143,7 +143,7 @@
 					<div class="tsd-signature tsd-kind-icon">Audio<wbr>Player<wbr>Playback<wbr>Stopped<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span><span class="tsd-signature-symbol"> =&nbsp;&quot;AudioPlayer.PlaybackStopped&quot;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L34">client/bst-alexa.ts:34</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L34">client/bst-alexa.ts:34</a></li>
 						</ul>
 					</aside>
 					<div class="tsd-comment tsd-typography">
@@ -159,7 +159,7 @@
 					<div class="tsd-signature tsd-kind-icon">Response<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span><span class="tsd-signature-symbol"> =&nbsp;&quot;response&quot;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-alexa.ts#L41">client/bst-alexa.ts:41</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-alexa.ts#L41">client/bst-alexa.ts:41</a></li>
 						</ul>
 					</aside>
 					<div class="tsd-comment tsd-typography">

--- a/docs/api/classes/bstencode.html
+++ b/docs/api/classes/bstencode.html
@@ -115,7 +115,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-encode.ts#L43">client/bst-encode.ts:43</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-encode.ts#L43">client/bst-encode.ts:43</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -151,7 +151,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-encode.ts#L63">client/bst-encode.ts:63</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-encode.ts#L63">client/bst-encode.ts:63</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -206,7 +206,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-encode.ts#L73">client/bst-encode.ts:73</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-encode.ts#L73">client/bst-encode.ts:73</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -267,7 +267,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-encode.ts#L91">client/bst-encode.ts:91</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-encode.ts#L91">client/bst-encode.ts:91</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -325,7 +325,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-encode.ts#L101">client/bst-encode.ts:101</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-encode.ts#L101">client/bst-encode.ts:101</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">

--- a/docs/api/classes/lambdaserver.html
+++ b/docs/api/classes/lambdaserver.html
@@ -111,7 +111,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/lambda-server.ts#L25">client/lambda-server.ts:25</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/lambda-server.ts#L25">client/lambda-server.ts:25</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -163,7 +163,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/lambda-server.ts#L40">client/lambda-server.ts:40</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/lambda-server.ts#L40">client/lambda-server.ts:40</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -206,7 +206,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/lambda-server.ts#L96">client/lambda-server.ts:96</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/lambda-server.ts#L96">client/lambda-server.ts:96</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">

--- a/docs/api/classes/log.html
+++ b/docs/api/classes/log.html
@@ -111,7 +111,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L270">logless/logless-context.ts:270</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L270">logless/logless-context.ts:270</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -142,7 +142,7 @@
 					<div class="tsd-signature tsd-kind-icon">_timestamp<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">Date</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L270">logless/logless-context.ts:270</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L270">logless/logless-context.ts:270</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -152,7 +152,7 @@
 					<div class="tsd-signature tsd-kind-icon">data<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">any</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L272">logless/logless-context.ts:272</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L272">logless/logless-context.ts:272</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -162,7 +162,7 @@
 					<div class="tsd-signature tsd-kind-icon">stack<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L272">logless/logless-context.ts:272</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L272">logless/logless-context.ts:272</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -172,7 +172,7 @@
 					<div class="tsd-signature tsd-kind-icon">tags<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">Array</span><span class="tsd-signature-symbol">&lt;</span><span class="tsd-signature-type">string</span><span class="tsd-signature-symbol">&gt;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L272">logless/logless-context.ts:272</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L272">logless/logless-context.ts:272</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -182,7 +182,7 @@
 					<div class="tsd-signature tsd-kind-icon">type<span class="tsd-signature-symbol">:</span> <a href="../enums/logtype.html" class="tsd-signature-type">LogType</a></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L272">logless/logless-context.ts:272</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L272">logless/logless-context.ts:272</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -199,7 +199,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L276">logless/logless-context.ts:276</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L276">logless/logless-context.ts:276</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">string</span></h4>

--- a/docs/api/classes/logless.html
+++ b/docs/api/classes/logless.html
@@ -144,7 +144,7 @@
 					<div class="tsd-signature tsd-kind-icon">Domain<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span><span class="tsd-signature-symbol"> =&nbsp;&quot;logless.bespoken.tools&quot;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless.ts#L49">logless/logless.ts:49</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless.ts#L49">logless/logless.ts:49</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -161,7 +161,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless.ts#L58">logless/logless.ts:58</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless.ts#L58">logless/logless.ts:58</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -195,7 +195,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless.ts#L122">logless/logless.ts:122</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless.ts#L122">logless/logless.ts:122</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">void</span></h4>
@@ -212,7 +212,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless.ts#L117">logless/logless.ts:117</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless.ts#L117">logless/logless.ts:117</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -237,7 +237,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless.ts#L72">logless/logless.ts:72</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless.ts#L72">logless/logless.ts:72</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -266,7 +266,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless.ts#L81">logless/logless.ts:81</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless.ts#L81">logless/logless.ts:81</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">

--- a/docs/api/classes/loglesscontext.html
+++ b/docs/api/classes/loglesscontext.html
@@ -110,7 +110,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L10">logless/logless-context.ts:10</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L10">logless/logless-context.ts:10</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -136,7 +136,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L125">logless/logless-context.ts:125</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L125">logless/logless-context.ts:125</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">Function</span></h4>
@@ -153,7 +153,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L73">logless/logless-context.ts:73</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L73">logless/logless-context.ts:73</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">
@@ -181,7 +181,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L180">logless/logless-context.ts:180</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L180">logless/logless-context.ts:180</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">void</span></h4>
@@ -198,7 +198,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L184">logless/logless-context.ts:184</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L184">logless/logless-context.ts:184</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -233,7 +233,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L129">logless/logless-context.ts:129</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L129">logless/logless-context.ts:129</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -265,7 +265,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L156">logless/logless-context.ts:156</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L156">logless/logless-context.ts:156</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -294,7 +294,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L83">logless/logless-context.ts:83</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L83">logless/logless-context.ts:83</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -323,7 +323,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L176">logless/logless-context.ts:176</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L176">logless/logless-context.ts:176</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">string</span></h4>
@@ -340,7 +340,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L220">logless/logless-context.ts:220</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L220">logless/logless-context.ts:220</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -378,7 +378,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L59">logless/logless-context.ts:59</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L59">logless/logless-context.ts:59</a></li>
 								</ul>
 							</aside>
 							<div class="tsd-comment tsd-typography">

--- a/docs/api/classes/loglessmiddleware.html
+++ b/docs/api/classes/loglessmiddleware.html
@@ -102,7 +102,7 @@
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless.ts#L149">logless/logless.ts:149</a></li>
+									<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless.ts#L149">logless/logless.ts:149</a></li>
 								</ul>
 							</aside>
 							<h4 class="tsd-parameters-title">Parameters</h4>
@@ -127,7 +127,7 @@
 					<div class="tsd-signature tsd-kind-icon">error<wbr>Handler<span class="tsd-signature-symbol">:</span> <a href="../interfaces/expresserrorhandler.html" class="tsd-signature-type">ExpressErrorHandler</a></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless.ts#L150">logless/logless.ts:150</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless.ts#L150">logless/logless.ts:150</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -137,7 +137,7 @@
 					<div class="tsd-signature tsd-kind-icon">request<wbr>Handler<span class="tsd-signature-symbol">:</span> <a href="../interfaces/expressrequesthandler.html" class="tsd-signature-type">ExpressRequestHandler</a></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless.ts#L150">logless/logless.ts:150</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless.ts#L150">logless/logless.ts:150</a></li>
 						</ul>
 					</aside>
 				</section>

--- a/docs/api/enums/logtype.html
+++ b/docs/api/enums/logtype.html
@@ -86,7 +86,7 @@
 					<div class="tsd-signature tsd-kind-icon">DEBUG<span class="tsd-signature-symbol">:</span> </div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L263">logless/logless-context.ts:263</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L263">logless/logless-context.ts:263</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -96,7 +96,7 @@
 					<div class="tsd-signature tsd-kind-icon">ERROR<span class="tsd-signature-symbol">:</span> </div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L264">logless/logless-context.ts:264</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L264">logless/logless-context.ts:264</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -106,7 +106,7 @@
 					<div class="tsd-signature tsd-kind-icon">INFO<span class="tsd-signature-symbol">:</span> </div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L265">logless/logless-context.ts:265</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L265">logless/logless-context.ts:265</a></li>
 						</ul>
 					</aside>
 				</section>
@@ -116,7 +116,7 @@
 					<div class="tsd-signature tsd-kind-icon">WARN<span class="tsd-signature-symbol">:</span> </div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless-context.ts#L266">logless/logless-context.ts:266</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless-context.ts#L266">logless/logless-context.ts:266</a></li>
 						</ul>
 					</aside>
 				</section>

--- a/docs/api/interfaces/expresserrorhandler.html
+++ b/docs/api/interfaces/expresserrorhandler.html
@@ -79,7 +79,7 @@
 					<li class="tsd-description">
 						<aside class="tsd-sources">
 							<ul>
-								<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless.ts#L192">logless/logless.ts:192</a></li>
+								<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless.ts#L192">logless/logless.ts:192</a></li>
 							</ul>
 						</aside>
 						<h4 class="tsd-parameters-title">Parameters</h4>

--- a/docs/api/interfaces/expressrequesthandler.html
+++ b/docs/api/interfaces/expressrequesthandler.html
@@ -79,7 +79,7 @@
 					<li class="tsd-description">
 						<aside class="tsd-sources">
 							<ul>
-								<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless.ts#L188">logless/logless.ts:188</a></li>
+								<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless.ts#L188">logless/logless.ts:188</a></li>
 							</ul>
 						</aside>
 						<h4 class="tsd-parameters-title">Parameters</h4>

--- a/docs/api/interfaces/iencoderconfig.html
+++ b/docs/api/interfaces/iencoderconfig.html
@@ -94,7 +94,7 @@
 					<div class="tsd-signature tsd-kind-icon">access<wbr>Key<wbr>Id<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-encode.ts#L16">client/bst-encode.ts:16</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-encode.ts#L16">client/bst-encode.ts:16</a></li>
 						</ul>
 					</aside>
 					<div class="tsd-comment tsd-typography">
@@ -109,7 +109,7 @@
 					<div class="tsd-signature tsd-kind-icon">bucket<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-encode.ts#L13">client/bst-encode.ts:13</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-encode.ts#L13">client/bst-encode.ts:13</a></li>
 						</ul>
 					</aside>
 					<div class="tsd-comment tsd-typography">
@@ -124,7 +124,7 @@
 					<div class="tsd-signature tsd-kind-icon">filter<wbr>Volume<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">number</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-encode.ts#L26">client/bst-encode.ts:26</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-encode.ts#L26">client/bst-encode.ts:26</a></li>
 						</ul>
 					</aside>
 					<div class="tsd-comment tsd-typography">
@@ -140,7 +140,7 @@
 					<div class="tsd-signature tsd-kind-icon">secret<wbr>Access<wbr>Key<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/client/bst-encode.ts#L19">client/bst-encode.ts:19</a></li>
+							<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/client/bst-encode.ts#L19">client/bst-encode.ts:19</a></li>
 						</ul>
 					</aside>
 					<div class="tsd-comment tsd-typography">

--- a/docs/api/interfaces/lambdafunction.html
+++ b/docs/api/interfaces/lambdafunction.html
@@ -86,7 +86,7 @@
 					<li class="tsd-description">
 						<aside class="tsd-sources">
 							<ul>
-								<li>Defined in <a href="https://github.com/bespoken/bst/blob/ded4bcd/lib/logless/logless.ts#L156">logless/logless.ts:156</a></li>
+								<li>Defined in <a href="https://github.com/bespoken/bst/blob/01e5c28/lib/logless/logless.ts#L156">logless/logless.ts:156</a></li>
 							</ul>
 						</aside>
 						<div class="tsd-comment tsd-typography">

--- a/package.json
+++ b/package.json
@@ -67,7 +67,7 @@
     "precoverage": "node_modules/gulp/bin/gulp.js build",
     "coverage": "istanbul cover _mocha test/**/*-test.js -- -R spec && remap-istanbul -i ./coverage/coverage.json -o ./coverage/ts-report -t html && ./node_modules/.bin/remap-istanbul -i coverage/coverage.json -t lcovonly > coverage/lcov-ts.info",
     "coveralls": "coveralls < coverage/lcov-ts.info",
-    "preversion": "gulp docs && git add docs/api && git commit -m 'Updated generated API docs'",
+    "preversion": "gulp build && gulp docs && git add docs/api && git commit -m 'Updated generated API docs'",
     "postversion": "git push && git push --tags"
   },
   "types": "index.d.ts"

--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
   "description": "Tools for making Alexa development easier and more fun",
   "license": "SEE LICENSE IN LICENSE",
   "private": false,
-  "version": "0.9.30",
+  "version": "0.9.31",
   "bin": {
     "bst": "./bin/bst.js",
     "bst-server": "./bin/bst-server.js"

